### DISCUSSION TITLE
How to create dropdown button in ui.aggrid's cell?

### DISCUSSION THREAD

**Discussion by tz301** [zauberzeug/nicegui#2849] [Q&A]
### Question

As title described, when specific cell is clicked, the dropdown down button can be shown to interact more.

**Comment by tz301** [comment]
@falkoschindler Do you have any idea?

**Reply by falkoschindler** [reply]
I'm not sure if I understand correctly. Do you want to embed NiceGUI elements in an AG Grid cell? That's not possible, I'm afraid.
But maybe there's a way to add dropdown buttons to AG Grid cells. Have you checked their documentation?
